src/main/scala/strawman/collection/Seq.scala

```diff
@@ -26,6 +31,9 @@ trait SeqOps[+A, +CC[X], +C] extends Any
   with ArrayLike[A]
   with Equals {
 
+  protected[this] def c: C
+  protected[this] def seq: Seq[A]
```

Comment:
This is unfortunate. For reasons explained in #134, I had to introduce these two methods. I need more time to think of a better solution.